src/content/self-hosted/administration/configuration.mdx

```diff
@@ -1010,6 +1010,33 @@ wildcardCertificate:
       key: "ca.crt"
 ```
 
+### user
+
+#### serviceAccount
+
+The configured annotations and labels will be applied to each service account created by Okteto.
+These annotations and labels are additional to the ones already applied by Okteto.
+
+- `annotations`: Annotations applied to each service account generated by Okteto for user accounts.
```

Comment:
@pchico83 should we also use the `extraAnnotations` pattern here like we do in role bindings to signify that this doesn't replace the base ones?

Reply:
same for labels

Reply:
@rberrelleza you mean the key? this will imply re-opening the issue that was already approved and merged.

Reply:
@rberrelleza in the helm world, everyone is using `labels` and `annotations` for extra labels/annotations. We are already using it for extra labels/annotations in namespaces, or our okteto components

Reply:
@pchico83 I've seen both. Given that you can't replace the annotations and labels entirely, but add them, I think the extraLabels and extraAnnotations are a better name for the configuration. 

We already use that pattern on 'extraEnv' and 'extraRoleBindings' in our chart. Given that this is something new, I feel it is worth the extra engineering work to make it consistent before we expose it to customers.

Reply:
We use `labels/annotations` in `namespace` which is a very use case than this one but for namespaces:
https://www.okteto.com/docs/self-hosted/administration/configuration/#namespace
And we do the same for extra labels/annotations on every one of our components:
- https://www.okteto.com/docs/self-hosted/administration/configuration/#api
- https://www.okteto.com/docs/self-hosted/administration/configuration/#autoscaler
- https://www.okteto.com/docs/self-hosted/administration/configuration/#buildkit
- ...
Using `labels/Annotations` is more aligned to our current chart

Reply:
ok.